lib/screens/crypto/crypto_purchase_estimates.dart

```diff
@@ -0,0 +1,100 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_sample_integration/contants.dart';
+import 'package:inai_flutter_sdk/inai_flutter_sdk.dart';
+
+class ThemeColors {
+  static const Color bgPurple = Color(0xff7673dd);
```

Comment:
Please create extension class for the colours and import that colour to this class